flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStResourceContainer.java

```diff
@@ -190,7 +191,10 @@ public DBOptions getDbOptions() {
         // configured,
         //  fallback to local directory currently temporarily.
         if (remoteForStPath != null) {
-            flinkEnv = new FlinkEnv(remoteForStPath.toString(), forstFileSystem);
+            flinkEnv =
+                    new FlinkEnv(
+                            remoteForStPath.toString(),
+                            new StringifiedForStFileSystem(forstFileSystem));
```

Comment:
NIT: `forstFileSystem` and `StringifiedForStFileSystem` . We should be consistent in the variable names `ForS` or `Fors`.